The test should crash with a run time error now.
You should see only one runtime error message. Any repetition
of the error message indicates a problem in the threads system.
Running 4 thread(s) please wait... 

Run-time error 102
File error.icn; Line 46
numeric expected
offending value: &null
Traceback:
   main(list_1 = [])
   suma(25,1) from line 26 in error.icn
   {25 + &null} from line 46 in error.icn
